### ISSUE TITLE
0.31.0 compatibilty

### ISSUE THREAD

**Issue by sam0x17** [anykeyh/clear#142]
This is the first error that comes up, though there are likely others:

```
In lib/clear/src/clear/sql/connection_pool.cr:2:35

 2 | @@connections = {} of String => Channel::Buffered(DB::Database)
                                     ^----------------
Error: undefined constant Channel::Buffered
```

**Comment by anykeyh** [comment]
You can use `branch: v0.7.2` in your shard to make it works temporarily. 

Note than there's still a bug with pg array described [here](https://github.com/will/crystal-pg/issues/186)

**Comment by anykeyh** [comment]
Closed by #143
